package-lock.json

```diff
@@ -136,7 +137,6 @@
       "integrity": "sha512-e7jT4DxYvIDLk1ZHmU/m/mB19rex9sv0c2ftBtjSBv+kVM/902eh0fINUzD7UwLLNR+jU585GxUJ8/EBfAM5fw==",
       "dev": true,
       "license": "MIT",
-      "peer": true,
```

Comment:
note: this might be changing due to mismatched npm version, maybe do `npm install -g npm@latest`